pom.xml

```diff
@@ -2,7 +2,11 @@
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
-
+  <parent>
+    <groupId>org.fcrepo</groupId>
+    <artifactId>fcrepo</artifactId>
+    <version>4.3.1-SNAPSHOT</version>
+  </parent>
   <groupId>org.fcrepo.migration</groupId>
   <artifactId>migration-utils</artifactId>
   <version>4.2.1-SNAPSHOT</version>
```

Comment:
Please update this `version` to `4.3.1-SNAPSHOT`, as well as the two other instances of `4.2.1-SNAPSHOT` found in this file.